.github/workflows/pre-submit.yml

```diff
@@ -15,12 +20,11 @@ jobs:
         with:
           go-version: '1.17'
        
-      - name: Run tests for build-go
+      - name: Run tests for builder
```

Comment:
should we test their binaries build too?

Reply:
which binary?

Reply:
ho, I got it, you mean make a `go build`. I was assuming this was implicit by building the uni tests, but I'm probably wrong. I'll add that